esp/applications/sql2ecl/ldap_authorization_map.yaml

```diff
@@ -1,4 +1,5 @@
 ldap:
+   resourcesBasedn: ou=WsSql,ou=EspServices,ou=ecl
```

Comment:
@afishbeck I don't think we can hard code an OU anywhere in the code, unless it's clear to the admin how to override.
With the new LDAPEnvironment tool, the OU have to be considered dynamic.  There will be a configurable OU root (ou=hpccRoot perhaps?), and the cluster name as part of the OU specification.
ou=WsSql,ou=EspServices,ou=Insurance,ou=hpccRoot
ou=WsSql,ou=EspServices,ou=Fraud,ou=hpccRoot
ou=WsSql,ou=EspServices,ou=MotorVehicles,ou=hpccRoot

Reply:
It's not hard coded, this is just the default.  But if you want to remove default OU values because they can't be defaulted that's fine with me.  Just either do it in a separate PR, it needs it's own flavor of testing.